<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WSTG8BLD</class>
 <widget class="QWidget" name="WSTG8BLD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WSTG8BLD</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(128, 128, 128);</string>
  </property>
  <widget class="QLabel" name="lbSTGBT_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>32</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>ST8DMD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGB_1">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>32</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>9.70</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBU_1">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>32</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBU_2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>47</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBT_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>47</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>ST8SEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGB_2">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>47</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>99.10</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBU_3">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGB_3">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>98.60</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBT_3">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>ST8A</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBU_4">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>75</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBT_4">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>75</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>ST8B</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGB_4">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>75</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>99.60</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBT_5">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>108</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>ST8MA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGB_5">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>108</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>10.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBU_5">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>108</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>MA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBU_6">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>OHMS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGBT_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>ST80HMS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSTGB_6">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>10.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btCerrar">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>173</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 192, 192);</string>
   </property>
   <property name="text">
    <string>STG8 BLD</string>
   </property>
  </widget>
  <zorder>lbFondo</zorder>
  <zorder>lbSTGBT_1</zorder>
  <zorder>lbSTGB_1</zorder>
  <zorder>lbSTGBU_1</zorder>
  <zorder>lbSTGBU_2</zorder>
  <zorder>lbSTGBT_2</zorder>
  <zorder>lbSTGB_2</zorder>
  <zorder>lbSTGBU_3</zorder>
  <zorder>lbSTGB_3</zorder>
  <zorder>lbSTGBT_3</zorder>
  <zorder>lbSTGBU_4</zorder>
  <zorder>lbSTGBT_4</zorder>
  <zorder>lbSTGB_4</zorder>
  <zorder>lbSTGBT_5</zorder>
  <zorder>lbSTGB_5</zorder>
  <zorder>lbSTGBU_5</zorder>
  <zorder>lbSTGBU_6</zorder>
  <zorder>lbSTGBT_6</zorder>
  <zorder>lbSTGB_6</zorder>
  <zorder>btCerrar</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
